src/components/SQForm/SQFormCheckboxGroup.tsx

```diff
@@ -25,7 +25,7 @@ interface SQFormCheckboxGroupProps {
   /** Whether a selection in this group is required */
   isRequired?: boolean;
   /** Function to call on value change */
-  onChange?: (event: React.ChangeEvent<HTMLInputElement>) => void;
+  onChange?: CheckboxProps['onChange'];
```

Comment:
Love this

Reply:
I think this was part of the key. Ensuring that the change handlers our components accept were the same as what MUI wants and then informing Formik about the type of the change handler. Instead of trying to keep both libraries happy.